tests/components/sensibo/snapshots/test_select.ambr

```diff
@@ -0,0 +1,170 @@
+# serializer version: 1
+# name: test_select[load_platforms0][select.hallway_horizontal_swing-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'options': list([
+        'stopped',
+        'fixedleft',
+        'fixedcenterleft',
+      ]),
+    }),
+    'config_entry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
+    'disabled_by': None,
+    'domain': 'select',
+    'entity_category': None,
+    'entity_id': 'select.hallway_horizontal_swing',
+    'has_entity_name': True,
+    'hidden_by': None,
+    'icon': None,
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': None,
+    'original_icon': None,
+    'original_name': 'Horizontal swing',
+    'platform': 'sensibo',
+    'previous_unique_id': None,
+    'supported_features': 0,
+    'translation_key': 'horizontalswing',
+    'unique_id': 'ABC999111-horizontalSwing',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_select[load_platforms0][select.hallway_horizontal_swing-state]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'friendly_name': 'Hallway Horizontal swing',
+      'options': list([
+        'stopped',
+        'fixedleft',
+        'fixedcenterleft',
+      ]),
+    }),
+    'context': <ANY>,
+    'entity_id': 'select.hallway_horizontal_swing',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'stopped',
+  })
+# ---
+# name: test_select[load_platforms0][select.hallway_light-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'options': list([
+        'on',
+        'off',
+      ]),
+    }),
+    'config_entry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
```

Comment:
why isn't this a switch?

Reply:
Because it can also have a value which is `dim` (which isn't the case for this hvac device. You see it on line 122

Reply:
Probably but I think that would be confusing for users that if they have two devices, in one case it's a switch and in the other, it's a select. 

In reality I doubt many are using this but I have of course no idea.

Reply:
I think it would make sense to have, since also the services are more clear and stuff like voice works better on a switch with only 2 options

Reply:
I'll go ahead and merge this PR, as this discussion is good... but not really related to this PR at the same time 😉